### PR TITLE
fix(docker): workaround to a behavior change in setuptools.

### PR DESCRIPTION
**Pull request by grignards**
The version introducing the change is the 60.0.0.

See #5183.

-------

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link): No automated tests for docker.
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: tools_docker/Debian_New_Uwsgi/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ RUN apt-get install -qy $(cat "${PACKAGE_DIR}/run_backend_packages.list") \
 # - installs optional dependencies
 ####
 FROM base AS build_backend
+ENV SETUPTOOLS_USE_DISTUTILS stdlib
```
Comment (PhilippeAccorsi) [inline tools_docker/Debian_New_Uwsgi/Dockerfile:37]: missing `=` no ?

File: tools_docker/Debian_Uwsgi/Dockerfile [hunk 2]
```diff
@@ -35,6 +35,7 @@ RUN apt-get install -qy --no-install-recommends \
 # - installs optional dependencies
 ####
 FROM base AS build_backend
+ENV SETUPTOOLS_USE_DISTUTILS stdlib
```
Comment (PhilippeAccorsi) [inline tools_docker/Debian_Uwsgi/Dockerfile:38]: missing `=` no ?

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

File: tools_docker/Debian_New_Uwsgi/Dockerfile (hunk 1 above)
Comment (grignards) [reply tools_docker/Debian_New_Uwsgi/Dockerfile]: You're right… I fixed it.

File: tools_docker/Debian_Uwsgi/Dockerfile (hunk 2 above)
Comment (grignards) [reply tools_docker/Debian_Uwsgi/Dockerfile]: Fixed.

File: tools_docker/Debian_New_Uwsgi/Dockerfile (hunk 1 above)
Comment (grignards) [reply tools_docker/Debian_New_Uwsgi/Dockerfile]: In fact both work (https://docs.docker.com/engine/reference/builder/#env, "Alternative syntax"), but as other `ENV` variables are written with `=` it is a better idea to keep the same syntax everywhere

Comment (PhilippeAccorsi) [reply tools_docker/Debian_New_Uwsgi/Dockerfile]: ok. thanks for this information \o/

**Comment by PhilippeAccorsi** [comment]
Docker image created and test on local machine.

**Review by PhilippeAccorsi — APPROVED** [review]

**Pull request merged**